src/main/java/org/c4sg/controller/UserController.java

```diff
@@ -243,4 +243,51 @@ public String uploadResume(@ApiParam(value = "user Id", required = true) @PathVa
         return null;
         }
     }
+    
+    @CrossOrigin
+    @RequestMapping(value = "/{id}/resume", method = RequestMethod.DELETE)
+    @ApiOperation(value = "Deletes user resume")
+    public String deleteResume(@ApiParam(value = "User id to delete resume for", required = true)
+                                         @PathVariable("id") int id) {
+    	UserDTO u = userService.findById(id);
+    	File res = new File(userService.getResumeUploadPath(id));
+    	try {
+    		boolean del = res.delete();
+    		u.setResumeUrl(null);
+    		userService.saveUser(u);
+    		if (del) {
+    			return "Success";
```

Comment:
Use ResponseEntity.noContent().build(); in the return statement. You can take the method getApplicants in the UserController class as an example to help you.